mindjunkies/lecture/tests/test_models.py

```
@@ -1,81 +1,219 @@
+from django.core.exceptions import ValidationError
 from django.utils.text import slugify
+from model_bakery import baker
+from django.db.utils import IntegrityError
+import cloudinary
+from cloudinary.models import CloudinaryField
 
-from mindjunkies.accounts.models import User
-from mindjunkies.courses.models import Course, Module
-from mindjunkies.lecture.models import Lecture, LecturePDF, LectureVideo
 
+from mindjunkies.lecture.models import Lecture, LecturePDF, LectureVideo, LectureCompletion, LastVisitedModule
```

Ruff (0.8.2)

`mindjunkies.lecture.models.LecturePDF` imported but unused

Remove unused import

(F401)